src/python_pachyderm/experimental/mixin/pps.py

```diff
@@ -118,8 +121,19 @@ def list_job(
         jqFilter : str, optional
             A ``jq`` filter that can filter the list of jobs returned, only if
             `pipeline_name` is provided.
-        project_name : str
+        project_name : str, optional
```

Comment:
Should we name this `pipeline_project_name` to make it clear this arg is to help specify which pipeline we're filtering jobs on?

Reply:
I'm going to decide to leave it just to be consistent with all of the other methods that use "project_name", but this does raise a good point that these new apis are getting to be confusing.